lib/IDE/SourceEntityWalker.cpp

```diff
@@ -134,7 +139,7 @@ bool SemaAnnotator::walkToDeclPre(Decl *D) {
         if (!SEWalker.visitDeclarationArgumentName(PD->getArgumentName(), Loc,
                                                    VD)) {
           Cancelled = true;
-          return true;
+          return false;
```

Comment:
Heh. Nathan and I were bamboozled by this one as well. This is actually in a lambda where the true/false is reversed. I'm happy for that to be changed/something else, but the `false` here actually means "no error". Ie. the check below is:
```
if (ReportParamList(ParamList)) return false;
```

So this should be true as is, not false.

Reply:
But… why? Before anyone gets hit by this again, I’m inverting the return value of `ReportParamList`.

Reply:
Why indeed 😆. Sounds good to me!